runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -161,6 +161,10 @@ protected boolean fillFromResolved(final long maxGlobal,
         INCLUDE_STOP    /** Stop, but also include this address. */
     }
 
+    private long backpointerCount = 0L;
+
+    public long getBackpointerCount() {return backpointerCount;}
+
     protected boolean followBackpointers(final UUID streamId,
```

Comment:
This is not related to your work but please break this unreadable method into smaller more understandable methods with comments. The code I was looking at earlier is a few days old and this one looks different from that one. Could you please break this method into smaller more cohesive methods that are less bug prone and more digestible. Debugging these things in a production scenario is not easy.